### PR TITLE
Fix for media elements within nested iFrame elements

### PR DESCRIPTION
**Pull request by rioam2**
Thanks for creating this extension! I've found it very helpful for reviewing some of my class material before exams :)

I made my own edit to enable usage on sites where media elements are nested within iframe elements. My university uses embedded players, which was originally not recognized by your extension. 

The changes in this PR allow the extension to recursively detect media elements within nested iframes on the page.

Does this sound like a good addition to the extension?

### CODE REVIEW

File: content/main.js [hunk 1]
```diff
@@ -464,10 +464,21 @@ const inspectElement = async (element) => {
 }
 
 const inspectAllMediaElements = () => {
-  const elements = [
-    ...document.getElementsByTagName('video'),
-    ...document.getElementsByTagName('audio'),
-  ];
+  function getElementsRecursive(rootEl) {
+    // Get all media elements within this root element context
```
Comment (vantezzen) [inline content/main.js:468]:
```suggestion
    if (!rootEl) return [];
    
    // Get all media elements within this root element context
```

I noticed that I get errors on some pages where `frame.contentDocument` is undefined due to permission restrictions so I'd suggest adding this small check.

**Review by vantezzen — COMMENTED** [review]
Thank you for this contribution! The change looks great, I have one suggestion though:

File: content/main.js (hunk 1 above)
Comment (rioam2) [reply content/main.js:470]: Good catch :) I've committed your suggested edits!

**Review by vantezzen — APPROVED** [review]
Great, thank you for your contribution!

**Pull request merged**